content/en/docs/concepts/overview/working-with-objects/labels.md

```diff
@@ -52,7 +52,7 @@ If the prefix is omitted, the label Key is presumed to be private to the user. A
 
 The `kubernetes.io/` and `k8s.io/` prefixes are reserved for Kubernetes core components.
 
-Valid label values must be 63 characters or less and must be empty or begin and end with an alphanumeric character (`[a-z0-9A-Z]`) with dashes (`-`), underscores (`_`), dots (`.`), and alphanumerics between.
+Valid label values must be 63 characters or less and must not be empty or begin and end with an alphanumeric character (`[a-z0-9A-Z]`) with dashes (`-`), underscores (`_`), dots (`.`), and alphanumerics between.
```

Comment:
Hi @Abdullah-F . Thanks for opening this pull request.
I agree that the sentence is not entirely clear.
What do you think about splitting the sentence into a couple of parts.
Also, can label values (not keys) be empty or undefined?
Possible edit:
Valid label values must be 63 characters or less and contain alphanumeric characters (`[a-z0-9A-Z]`) , dashes (`-`), underscores (`_`), and dots (`.`). Label values can be empty.

Reply:
HI @Abdullah-F.
Do the review comments make sense?

Reply:
Are empty label values permitted? If so, this PR is inaccurate.

Reply:
The current content of https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set seems fairly clear.